app/src/main/java/com/weatherxm/util/Rewards.kt

```diff
@@ -6,13 +6,23 @@ import com.weatherxm.ui.common.AnnotationGroupCode
 
 object Rewards {
 
+    @Suppress("MagicNumber")
+    fun isQodError(score: Int?): Boolean {
+        return score != null && score < 20
+    }
+
+    @Suppress("MagicNumber")
+    fun isQodWarning(score: Int?): Boolean {
+        return score != null && score in 20..79
+    }
```

Comment:
My suggestion is to declare a function that returns an error type (warning, error) to keep things cleaner. You can keep these functions with a modified body (eg.  `warningType(score) == .error`)